setup.cfg

```diff
@@ -54,13 +54,12 @@ install_requires =
     pydicom-seg==0.4.1
     pynetdicom==2.0.2
     pynrrd==1.0.0
-    opencv-python-headless==4.9.0.80
```

Comment:
removing this will not help open source monailabel users to do pip-install and run monailabel